geode-core/src/main/java/org/apache/geode/management/internal/cli/functions/RegionCreateFunction.java

```diff
@@ -94,6 +94,12 @@ public void execute(FunctionContext context) {
     try {
       Region<?, ?> createdRegion = createRegion(cache, regionCreateArgs);
       XmlEntity xmlEntity = new XmlEntity(CacheXml.REGION, "name", createdRegion.getName());
+
+      Region<?, ?> rootRegion = createdRegion.getRootRegion();
+      if (rootRegion != null) {
```

Comment:
I'm not that familiar with this code but I'm wondering about the case when there are multiple parent regions (eg `root/parent1/parent2/subregion`).